### ISSUE TITLE
node@22.9.0 is not supported 

### ISSUE THREAD

**Issue by JustFly1984** [SocketDev/socket-cli-js#226]
Trying the example from twitter, it refuses to start in vite/remix project with latest package versions

```
Now using node v22.9.0 (npm v10.8.3)
❯ pnpx @socketsecurity/cli optimize
✘ The engines.node range is not supported.
```

```package.json
  "packageManager": "pnpm@9.12.2",
  "engines": {
    "node": ">=22.9.0",
    "pnpm": ">=9.12.2"
  },
  ```

**Comment by bcomnes** [comment]
Our field is set to "node": "^20.9.0 || >=22.0.0", I'm not sure why this would be the case.

**Comment by jdalton** [comment]
@JustFly1984  That error is related to a sniff we've done on browserslist entry in your package.json. Can you post that to help us better debug.

@bcomnes The error is coming from the `optimize` command. I released an update early this afternoon to make the error more clear.

**Comment by jdalton** [comment]
@JustFly1984 Fixed in `v0.14.8`. (update `@socketsecurity/cli` and try again 🚀 )

**Comment by JustFly1984** [comment]
I do not have browserlist record in package.json though

**Comment by jdalton** [comment]
Sorry about that. Just published `v0.14.10`

**Comment by jdalton** [comment]
I'm seeing pnpm lock not being read correctly. Will dig in and report back when resolved.

**Comment by JustFly1984** [comment]
pnpx still running 14.8

**Comment by jdalton** [comment]
Just published `v0.14.11`.

**Comment by JustFly1984** [comment]
ok, weird thing, pnpx does not running latest version, it still running 0.14.8
I had to `pnpm install @socketsecurity/cli -D` and add the script `"securitytoken": "pnpx socket optimize"` to the package.json to run the latest version, cos even after installing to devDependencies pnpx still running 0.14.8

After running optimize script I've got a bunch of overrides, each package has ^1 version.

Is it possible to specify exact override versions without the ^ sign? 
For me it is the end consumer project, so I want to be able to verify what exactly has been updated, each time I run optimize in the long term, and roll back if any issues with any of your packages goes wrong.

BTW after running the build in my project, looks like bundle size didn't change, so I'm not ure how usefull yourpackage for my case.
